components/__init__.py

```diff
@@ -342,11 +342,13 @@ def define_components(reqs):
     retriever = GitRepoRetriever("https://github.com/spdk/spdk.git", True)
     reqs.define('spdk',
                 retriever=retriever,
-                commands=['./configure --prefix="$SPDK_PREFIX" ' \
-                          ' --with-fio="$FIO_SRC"',
+                commands=['./configure --prefix="$SPDK_PREFIX" --with-shared',
+                          #' --with-fio="$FIO_SRC" ' TODO: re-enable fio_plugin
```

Comment:
Hmm.   One option here would be to define a variable (like FIO_SRC)

SPDK_OPT="--with-shared"

and use $SPDK_OPT here ?

A bit hacky but it would allow us to change it when we update to a version that supports fio again